validator/src/main.rs

```diff
@@ -2003,6 +2024,39 @@ pub fn main() {
             )
             .after_help("Note: the new filter only applies to the currently running validator instance")
         )
+        .subcommand(
+            SubCommand::with_name("set-relayer-config")
+                .about("Set configuration for connection to a relayer")
+                .arg(
+                    Arg::with_name("relayer_url")
+                        .long("relayer-url")
+                        .help("Relayer url. Set to empty string to disable relayer connection.")
```

Comment:
if they omit, should also disable relayer connection

Reply:
can't omit, it's required

i don't think we want it to disconnect if they issue sub command with no arguments, too easy to accidentally do

if they want to disconnect, need to purposefully set required parm to ""